packages/mask/src/plugins/Wallet/apis/debank.ts

```diff
@@ -23,13 +23,30 @@ export async function getAssetsList(address: string) {
     }
 }
 
+const auroraChainOnDebank = getDeBankConstants(ChainId.Aurora).CHAIN_ID
+const fieldKeys = ['fast', 'normal', 'slow'] as const
+/**
+ * Debank's data might be outdated, like gas price for aurora which requires 1 Gwei at least
+ * https://twitter.com/AlexAuroraDev/status/1490353255817302016
+ * Once debank fixes it, we will remove this modifier.
+ */
+function gasModifier(gasDict: GasPriceDictResponse, chain: string) {
+    if (chain === auroraChainOnDebank) {
+        fieldKeys.forEach((fieldKey) => {
+            const field = gasDict.data[fieldKey]
+            field.price = Math.max(field.price, 1000000000)
```

Comment:
This might be more intuitive

```suggestion
            field.price = Math.max(field.price, formatGweiToWei(1).toFixed())
```

Reply:
Good suggestion, thx.